.github/workflows/nightly_build.yml

```diff
@@ -346,7 +346,7 @@ jobs:
     - name: Build package
       run: |
         export TIMESTAMP=`date '+%Y%m%d'`
-        export PYPI_VERSION=2.3.0
+        export PYPI_VERSION=$(cat ./python/version.txt | head -1)
```

Comment:
this cat will get 2.3.0.dev0 right? this is wrong, for nightly build the version should be 2.3.0b20230317
We can keep this in nightly build, only make sure we use version.txt when we release

Reply:
ok, change `version.txt` in to `2.3.0`, restore changes in `nightly_build.yml`

Reply:
yes, change back and we can manually modify this?

Reply:
no, version.txt should always be the correct version, in main it should be dev0

Reply:
just change this file back to 2.3.0 and we can manually modify this